nomad/interfaces.go

```diff
@@ -1,11 +0,0 @@
-package nomad
```

Comment:
Note for review: this only supported the now-removed `nomad/core_sched_test.go` file.